pkg/buildermgr/envwatcher.go

```diff
@@ -181,7 +177,7 @@ func (envw *environmentWatcher) watchEnvironments() {
 func (envw *environmentWatcher) sync() {
 	maxRetries := 10
```

Comment:
I see a bunch of code removed, some commented but the method declaration kept as it is - can you please clean this up?